app/src/main/java/org/wikipedia/editactionfeed/AddTitleDescriptionsActivity.kt

```diff
@@ -95,9 +84,22 @@ class AddTitleDescriptionsActivity : SingleFragmentActivity<AddTitleDescriptions
             AlertDialog.Builder(context)
                     .setCustomTitle(DialogTitleWithImage(context, R.string.description_edit_task_unlock_title, R.drawable.ic_illustration_description_edit_trophy, true))
                     .setMessage(R.string.description_edit_task_unlock_body)
-                    .setPositiveButton(R.string.title_description_get_started) { _, _ -> context.startActivity(AddTitleDescriptionsActivity.newIntent(context, EDIT_TASKS_TITLE_DESC_SOURCE)) }
+                    .setPositiveButton(R.string.onboarding_get_started) { _, _ -> context.startActivity(AddTitleDescriptionsActivity.newIntent(context, EDIT_FEED_TITLE_DESC)) }
                     .setNegativeButton(R.string.onboarding_maybe_later, null)
                     .show()
         }
+
+        fun maybeShowTranslationEdit(context: Context) {
+            if (WikipediaApp.getInstance().language().appLanguageCodes.size > 1 && Prefs.getTotalUserDescriptionsEdited() >= 2 && Prefs.showEditActionTranslateDescriptionsUnlockedDialog()) {
```

Comment:
Is it possible to have the same way of expressing the logic in `maybeShowEditUnlockDialog`?
```
if(xxx) {
  return
}
```

Reply:
@cooltey I did not understand this clearly... do you mean rather than have multiple logic checks at the if statement move it inside the method? is it for readability?

Reply:
What I meant was having the same coding style with the code in line 77 to line 80.

```
if(xxx) {
  return
}
AlertDialog.Builder(context) ...
```

Reply:
Got it! Done..